### ISSUE TITLE
Contributing instructions on wiki are out of date

### ISSUE THREAD

**Issue by coaxial** [maid/maid#202]
Here: https://github.com/maid/maid/wiki/Contributing

It recommends using Vagrant + Ubuntu 12.04. It might make more sense to now recommend using whatever OS the contributor prefers and use RVM with a gemset.

**Comment by benjaminoakes** [comment]
Just a quick warning: I gave this recommendation after getting burned by some specs that modified files on my system. Nothing bad, but that's why I started preferring some kind of virtualization.  For your consideration. 🙂

**Comment by coaxial** [comment]
Yeah, I had this thought afterwards as well. The test suite runs under FakeFS but you never know... Vagrant instructions (and the `Vagrantfile`) would need updates in that case.

**Comment by benjaminoakes** [comment]
Yeah, definitely. Some kind of " jail " seems necessary, just in case

**Comment by benjaminoakes** [comment]
A ton has changed in the 10 years since this was originally written, so I would imagine a lot would need to change without even reviewing the code

**Comment by coaxial** [comment]
I have updated the Vagrantfile so it can run with the project's target version and a more recent Ubuntu, along with instructions on how to run the tests in vagrant. It does compile ruby from source though (I guess it can't find precompiled binaries?) so `vagrant up` takes a little while.

I'm hoping that we can eventually avoid the risk of writing to the live filesystem altogether by refactoring the code, but I need to do more thinking before that happens.

In the meantime, `fakefs: true` at the `describe` level avoids writing to the live filesystem. I have updated the "Contributing" page on the wiki accordingly.
